scapy/utils.py

```diff
@@ -1124,7 +1124,11 @@ def wireshark(pktlist):
     """Run wireshark on a list of packets"""
     f = get_temp_file()
     wrpcap(f, pktlist)
-    subprocess.Popen([conf.prog.wireshark, "-r", f])
+    try:
+        subprocess.Popen([conf.prog.wireshark, "-r", f])
+    except OSError as e:
+        msg = "wireshark(): executing '%s' failed with '%s'"
+        log_scapy.error(msg % (conf.prog.wireshark, e))
```

Comment:
You should not format the message here but let `.error()` format it for you. Also, you can use `exc_info` to add the exception information to the logged message:

```Python
    except OSError:
        log_scapy.error("wireshark(): executing %r failed", conf.prog.wireshark, exc_info=1)
```
What do you think?